src/OnboardingSPA/components/Drawer/DrawerPanel/DesignColors.js

```diff
@@ -1,15 +1,318 @@
+
+		return (
+			<div className='custom-palette'>
+				<div className='custom-palette__top'
+					onClick={(e) => setIsAccordionClosed(!isAccordionClosed)}>
+					<div className='custom-palette__top-text'>SELECT CUSTOM COLORS</div>
+					{isAccordionClosed && <div className='custom-palette__top-icon'>+</div> }
+					{!isAccordionClosed && <div className='custom-palette__top-icon'>-</div> }
+				</div>
+				<div className={`custom-palette__below ${isAccordionClosed ? 'custom-palette_acc_closed' : 'custom-palette_acc_opened' }`}>
+					<div className='custom-palette__below-row'
+						onClick={(e) => selectCustomColor('background')}>
+						<div className={`custom-palette__below-row-icon ${customColors?.background && 'custom-palette__below-row-icon_selected_border'}`}
+							style={{ backgroundColor: `${customColors?.background ?? '#FFF'}` }}>
+							{customColors?.background ? <div>&#10003;</div> : null}
+							</div>
+						<div className='custom-palette__below-row-text'>Background</div>
+					</div>
+					<div className='custom-palette__below-row'
+						onClick={(e) => selectCustomColor('primary')}>
+						<div className={`custom-palette__below-row-icon ${customColors?.primary && 'custom-palette__below-row-icon_selected_border'}`}
+							style={{ backgroundColor: `${primaryColorTemp}` }}>
+							{customColors?.primary ? <>&#10003;</> : null}
+						</div>
+						<div className='custom-palette__below-row-text'>Primary</div>
+					</div>
+					<div className='custom-palette__below-row'
+						onClick={(e) => selectCustomColor('secondary')}>
+						<div className={`custom-palette__below-row-icon ${customColors?.secondary && 'custom-palette__below-row-icon_selected_border'}`}
+							style={{ backgroundColor: `${secondaryColorTemp}` }}>
+							{customColors?.secondary ? <>&#10003;</> : null}
+							</div>
+						<div className='custom-palette__below-row-text'>Secondary</div>
+					</div>
+					<div className='custom-palette__below-row'
+						onClick={(e) => selectCustomColor('tertiary')}>
+						<div className={`custom-palette__below-row-icon ${customColors?.tertiary && 'custom-palette__below-row-icon_selected_border'}`}
+							style={{ backgroundColor: `${tertiaryColorTemp}` }}>
+							{customColors?.tertiary ? <>&#10003;</> : null}
+							</div>
+						<div className='custom-palette__below-row-text'>Tertiary</div>
+					</div>
+				</div>
+				{showColorPicker && (
+					<Popover>
+						<div className='custom-palette__picker-close-icon'
+							onClick={() => setShowColorPicker(false)}>X</div>
+						<ColorPicker
+							onChange={changeCustomPickerColor}
+							defaultValue="#874141"
+						/>
+					</Popover>
 				)}
-			</p>
+			</div>
+		);
+	}
+
+	return (
+		<div style={{ padding: '0 4px' }}>
+			<h2>{__('Color Palettes', 'wp-module-onboarding')}</h2>
+			{ colorPalettes && buildPalettes()}
+			{buildCustomPalette()}
```

Comment:
Let's wait for `colorPalettes` in both the function calls? It looks like the `custom` color picker will render first then will get pushed down by the `theme` color palettes once the API returns them, this will look confusing on a mobile device.